etl/dca_operatingbusinesses.py

```diff
@@ -29,6 +29,14 @@ def get_address(hnum, sname):
     filter_rows(not_equals = [
             dict(address_borough = 'Outside NYC'),
             ]),
+
+    filter_rows(equals = [
+            dict(industry = 'Scrap Metal Processor'),
+            dict(industry = 'Parking Lot'),
```

Comment:
this filter can be done in postgres

Reply:
I discussed with Amanda, it's more efficient to conduct filter here, since only 5k records will be remained out of 180k